dashboard/client/requests_over_time/requests_over_time.html

```diff
@@ -0,0 +1,7 @@
+<template name="requestsOverTime">
+  <h2>Requests over time</h2>
+
```

Comment:
In this case, the blank line separates two block elements. Leaving a blank line between blocks improves readability. This is similar to how we leave blank lines between statements in our codebase.